packages/manager/apps/dedicated/client/app/dedicated/server/terminate/dedicated-server-terminate.controller.js

```diff
@@ -35,6 +37,8 @@ angular.module('App').controller(
       this.$scope.submitCancelSubscription = this.submitCancelSubscription.bind(
         this,
       );
+      this.terminationUnavailable =
+        this.$scope.user.isEnterprise && this.coreConfig.isRegion('US');
```

Comment:
Maybe we could use feature flipping for that ?

Reply:
We will still need this feature for `US`. It is just for the enterprise users, that we need to disable the feature. I guess this is not possible right now using `feature-flipping`

Reply:
Maybe we could add a rule like `isEnterprise` in `feature-flipping`?

Reply:
This is done. Thanks!